packages/odyssey-react/src/components/List/List.module.scss

```diff
@@ -45,7 +45,7 @@
 
 .description {
   display: grid;
-  grid-gap: $spacing-xs $spacing-m;
+  grid-gap: var(--DescriptionGridGapRows) var(--DescriptionGridGapColumns);
```

Comment:
consider moving away from the shorthand here to help make the CSS easier to parse with variables involved *and* cleanup the variable names themselves too:

`row-gap`
`column-gap`

Reply:
updated in e9bb90c